test/Microsoft.Extensions.Configuration.EnvironmentVariables.Test/EnvironmentVariablesTest.cs

```diff
@@ -146,8 +146,8 @@ public void ReplaceDoubleUnderscoreInEnvironmentVariables()
 
             envConfigSrc.Load(dict);
 
-            Assert.Equal("connection", envConfigSrc.Get("data:ConnectionString"));
-            Assert.Equal("System.Data.SqlClient", envConfigSrc.Get("Data:_db1:ProviderName"));
+            Assert.Equal("connection", envConfigSrc.Get("ConnectionString"));
```

Comment:
This Assert should really not change as it doesn't match the format of an Azure Web Sites connection string.